configs/defaults/large_cohort.toml

```diff
@@ -14,6 +14,17 @@ highmem_workers = true
 # Calling intervals (defauls to whole genome intervals)
 #intervals_path =
 
+# Override for QoB JAR to fix teh StreamConstraints issue
+jar_spec_revision = 'f5b65edb8f1ed8e431f1799546a8e96d0ddff7d9'
```

Comment:
I'd strongly recommend not checking this in. In particular, [this one is now 404](https://centrepopgen.slack.com/archives/C030X7WGFCL/p1734320188057349?thread_ts=1732046164.229849&cid=C030X7WGFCL).

Reply:
Yes, was just in the process of checking that I had the correct jar. Will update

Reply:
Okay, I'll just leave it as: I'd strongly recommend not checking this in.

Reply:
Echoing John's comment, I don't think we want to hard code _any_ jar_spec_revision in. This is going to change over time, and we want the person running it to have to actually make a deliberate decision.

Reply:
Fair points, I will take it out so that it has to be specified by the user.